src/tools/cephfs/shell/cephfs-shell

```diff
@@ -1100,7 +1103,16 @@ class CephFSShell(Cmd):
                     path) if is_file_exists(i)])
             else:
                 try:
-                    cephfs.unlink(path)
+                    if is_dir_exists(path) and args.recursive:
+                        files = list(sorted(set(dirwalk(path)), reverse=True))
+                        for filepath in files:
```

Comment:
nit: I would say let's keep loop variable and variable being looped with similar names.
```suggestion
                        filepaths = list(sorted(set(dirwalk(path)), reverse=True))
                        for filepath in files:
```